synapse/cores/lmdb.py

```diff
@@ -515,7 +640,9 @@ def _getSizeByProp(self, prop, valu=None, limit=None, mintime=None, maxtime=None
 
     def _getRowsByProp(self, prop, valu=None, limit=None, mintime=None, maxtime=None,
                        do_count_only=False):
-        indx = self.index_pt if valu is None else self.index_pvt
+        indx = self._dbs.get(PROP_VAL_INDEX)
```

Comment:
Since we don't have a good performance testing harness setup yet, how did you go about measuring the per function performance here?  The _dbs refactoring is probably revertable without to much effort; esp if the perf impact is significant.  Even then, if we're that perf sensitive, it is probably best to utilize single character class names to speed up the hashing required when doing attribute (which goes through self.__dict__).